alchemy/src/cloudflare/bucket-object.ts

```diff
@@ -1,4 +1,3 @@
-import type { R2PutOptions } from "@cloudflare/workers-types/experimental/index.ts";
```

Comment:
Removed because it was causing a type error. Now it relies on the global from @cloudflare/workers-types.